grazer/src/main/java/edu/asu/diging/grazer/web/FileUploadController.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.grazer.web;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+import edu.asu.diging.grazer.core.domain.impl.FileTransformationImpl;
+import edu.asu.diging.grazer.core.fileupload.service.impl.FileUploadServiceImpl;
+
+@Controller
+public class FileUploadController {
+    
+    @Autowired
+    private FileUploadServiceImpl service;
+    
+    @RequestMapping(value = "/transformation/save", method = RequestMethod.POST)
+    public String uploadResources(@ModelAttribute FileTransformationImpl transformation,
```

Comment:
you probably want to use a validator here to make sure two files are uploaded and a label is set (use spring validation).